recipes/lightpath/LICENSE.md

```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2018, The Board of Trustees of the Leland Stanford Junior
```

Comment:
The `LICENSE.md` is included in the repository (https://github.com/pcdshub/lightpath/blob/master/LICENSE.md) so it doesn't need to be included here.